extendr-api/src/wrapper/logicals.rs

```diff
@@ -41,6 +41,21 @@ impl Logicals {
     }
 }
 
+impl TryFrom<Vec<i32>> for Logicals {
+    type Error = Error;
+
+    fn try_from(value: Vec<i32>) -> std::result::Result<Self, Self::Error> {
+        let value_rbools: std::result::Result<Vec<Rbool>, _> = value
+            .into_iter()
+            .map(|v| Rbool::try_from_strict(v))
```

Comment:
We want the strict try_from here because if it can't convert _everything_ into Rbool we want Err? 
What's somewhat frustrating is that I would want `From<Vec<i32>> for Logicals` to behave like `from_values()` but we can't have that, right? They would be conflicting implementations?

Reply:
I know it is too much to ask for.. But can you make an overview of the choices? Because @Ilia-Kosenkov said https://github.com/extendr/extendr/pull/596#issuecomment-1701648049, but
`from_values`, uses the current `From<Rbool>` rule, which is.. I'mma guess not the same as the proposed version with @Ilia-Kosenkov 's

Reply:
At this point, I have no strong opinion. It could be either `From` using the logic I mentioned (C-like), or it could be `TryFrom`, which would be more 'precise'. Note that in the case of `From`, we ensure that the resulting `Rbool` is either `1`, `0`, or `NA` regardless of the `i32` input.

Reply:
so.. Merge this?